gtfs2netexfr/README.md

```diff
@@ -0,0 +1,23 @@
+`gtfs2netexfr`
+=====
+
+Command-Line Interface to convert [GTFS] data format into [NeTEx] (profile
+France) data format.
+
+[GTFS]: https://developers.google.com/transit/gtfs/reference/
+[NeTEx]: http://netex-cen.eu/
+
+# Installation
+To install, use
+
+```bash
+cargo install gtfs2netexfr
+```
+
+# Usage
+
+```bash
+gtfs2netexfr --input /path/to/gtfs/folder/ --ouput /path/to/netexfr/
```

Comment:
isn't there also the mandatory `--participant` ? (it could help to explain a bit this parameter, maybe here and in the cli documentation)